src/server/api/util/api.js

```diff
@@ -132,7 +133,7 @@ const getOAuthURL = ({ state, isRegister, from, referrer }) => {
   const qsPayload = {
     client_id: LIKE_CO_CLIENT_ID,
     redirect_uri: OAUTH_REDIRECT_URI,
-    scope: 'profile email read:like.info read:civic_liker write:civic_liker',
+    scope: OAUTH_SCOPE_REQUEST.join(' '),
```

Comment:
`OAUTH_SCOPE_REQUEST` or `OAUTH_SCOPE_REQUIRED`? Seems the original one is `OAUTH_SCOPE_REQUIRED`

Reply:
`REQUEST` because we have a new feature that auto spread scope without `read` or `write` into both `read` and `write`